templates/aws/index.html

```diff
@@ -291,4 +291,16 @@ <h2>Optimise your cloud journey with Canonical</h2>
     </div>
   </section>
 
+  <!-- Set default Marketo information for contact form below-->
+  <div class="u-hide" 
+    id="contact-form-container" 
+    data-form-location="/shared/forms/form-template" 
+    data-form-id="3790" 
+    data-lp-id="{{ formData.formId }}" 
```

Comment:
No idea either, I'll remove it since it is empty in the [original form](https://github.com/canonical/ubuntu.com/blob/main/templates/aws/base_aws.html)